src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -294,6 +294,21 @@
             "ClientsSupportedPerFabric",
             "MaximumCheckInBackOff"
         ],
+        "Network Commissioning": [
+            "MaxNetworks",
+            "Networks",
+            "ScanMaxTimeSeconds",
```

Comment:
The implementation sometimes falls through for ScanMaxTimeSeconds, so it's not always handled via AttributeAccessInterface.  Can't add it here until it is.

Reply:
Should I change the implementation so it can't fallthrough. All platforms in the repo implement their NWC driver which provides a getter for ScanMaxTimeSeconds and ConnectMaxTimeSeconds from what I can see.

Reply:
I have no opinions on what the implementation should do, honestly.  Either it should always encode-a-value-or-error, and then we can add the thing here, or it should not, and then we can't add the thing here.